yarn.lock

```diff
@@ -10646,7 +10620,6 @@ sass@1.26.5, sass@^1.23.0:
 
 "sauce-connect-proxy@https://saucelabs.com/downloads/sc-4.5.4-linux.tar.gz":
   version "0.0.0"
-  uid dc5efcd2be24ddb099a85b923d6e754754651fa8
```

Comment:
any idea why this changed?

Reply:
No idea, I just ran `yarn install` and the `yarn.lock` updated